docs/sources/visualizations/time-series/index.md

```diff
@@ -0,0 +1,266 @@
+---
+aliases:
+  - /docs/grafana/latest/panels/visualizations/time-series/
+  - /docs/grafana/latest/visualizations/time-series/
+  - /docs/grafana/latest/panels/visualizations/time-series/annotate-time-series/
+  - /docs/grafana/latest/visualizations/time-series/annotate-time-series/
+  - /docs/grafana/latest/panels/visualizations/time-series/change-axis-display/
+  - /docs/grafana/latest/visualizations/time-series/change-axis-display/
+  - /docs/grafana/latest/panels/visualizations/time-series/graph-color-scheme/
+  - /docs/grafana/latest/visualizations/time-series/graph-color-scheme/
+  - /docs/grafana/latest/panels/visualizations/time-series/graph-time-series-as-bars/
+  - /docs/grafana/latest/visualizations/time-series/graph-time-series-as-bars/
+  - /docs/grafana/latest/panels/visualizations/time-series/graph-time-series-as-lines/
+  - /docs/grafana/latest/visualizations/time-series/graph-time-series-as-lines/
+  - /docs/grafana/latest/panels/visualizations/time-series/graph-time-series-as-points/
+  - /docs/grafana/latest/visualizations/time-series/graph-time-series-as-points/
+  - /docs/grafana/latest/features/panels/histogram/
+  - /docs/grafana/latest/panels/visualizations/time-series/graph-time-series-stacking/
+  - /docs/grafana/latest/visualizations/time-series/graph-time-series-stacking/
+keywords:
+  - grafana
+  - graph panel
+  - time series panel
+  - documentation
+  - guide
+  - graph
+title: Time series
+weight: 1200
+---
+
+# Time series
+
+{{< figure src="/static/img/docs/time-series-panel/time_series_small_example.png" max-width="1200px" caption="Time series" >}}
+
+Time series visualization is the default and primary way to visualize time series data. It can render as a line, a path of dots, or a series of bars. Time series visualizations are versatile enough to display almost any time-series data. [This public demo dashboard](https://play.grafana.org/d/000000016/1-time-series-graphs?orgId=1) contains many different examples for how time series visualizations can be configured and styled.
+
+> **Note:** You can migrate Graph panel visualizations to Time series visualizations. To migrate, open the panel and select the **Time series** visualization. Grafana transfers all applicable settings.
```

Comment:
```suggestion
The time series visualization type is the default and primary way to visualize time series data in a graph. It can render series as lines, points, or bars. It is versatile enough to display almost any time-series data. [This public demo dashboard](https://play.grafana.org/d/000000016/1-time-series-graphs?orgId=1) contains many different examples of how it can be configured and styled.

> **Note:** You can migrate from the old Graph visualization to the new Time series visualization. To migrate, open the panel and click the "Migrate" button in the side pane.
```

`Time series visualization` is a bit of a long name to repeat, (and at one place it is in plural where it should be singular)